airgun/widgets.py

```diff
@@ -727,8 +727,9 @@ class SatFlashMessages(FlashMessages):
 
     """
 
```

Comment:
Update the docstring with example as well.

Reply:
How about this  for doc string:

+        <ul class="pf-c-alert-group pf-m-toast"><li>
+        <div class="pf-c-alert pf-m-success foreman-toast" aria-label="Success Alert"
+            data-ouia-component-type="PF4/Alert" data-ouia-safe="true"
+            data-ouia-component-id="OUIA-Generated-Alert-success-1">
+            <div class="pf-c-alert__icon"

     Locator example::
 
-        //div[@class="toast-notifications-list-pf"]
+        //ul[@class="pf-c-alert-group pf-m-toast"]/li/div[contains(@class, "pf-c-alert")]